docs/ARCHITECTURE.md

```diff
@@ -9,10 +23,34 @@ can be configured to run only during certain time periods or not at all. Setting
 make sure repair is spread out over the interval time while alarms are provided to signal when a job has not run for
 longer than expected.
 
+<figure>
+  <div align="center">
+    <img src="images/ecchronos-instances.png" alt="Alt Text" style="display: block; margin-left: auto; margin-right: auto; width: 500px;">
+    <br/>
+    <figcaption >Figure 1: ecChronos and Cassandra Nodes.</figcaption>
+  </div>
+</figure>
+
 ## Concepts
 
 ### Leases
 
+In the context of Apache Cassandra, a "lease" refers to a mechanism employed during the repair process within Cassandra.
+
+Upon the initiation of a repair in a Cassandra cluster, a "lease" is granted to one of the cluster nodes, which serves as the repair coordinator. This coordinating node is responsible for overseeing the repair process to ensure that all data replicas conform and any disparities are addressed.
```

Comment:
Why the use of "lease" everywhere? Why not just use lease (without citation) if that's the correct word?